scripts/dist.ts

```diff
@@ -238,12 +238,12 @@ function toDist(sourcePath: string): YAML.Document {
 
     if (!source.draft_date) {
       logger.error(
-        `${id}: contains at least one deprecated compat feature and can never be Baseline. This is forbidden for published features.`,
+        `${id}: contains at least one deprecated compat feature. This is forbidden for published features.`,
```

Comment:
This is kinda annoying in some cases, because you never get the one key that caused it. I should refactor this a bit, to conditionally check deprecated for each key up here.